mipidsi/src/models.rs

```diff
@@ -35,6 +47,9 @@ pub trait Model {
     /// The framebuffer size in pixels.
     const FRAMEBUFFER_SIZE: (u16, u16);
 
+    /// Endianness expectation of the display Model's data, defaults to big endian
+    const ENDIANNESS: Endianness = Endianness::BigEndian;
```

Comment:
Are there any little endian MIPI DCS displays? If not, this is just pointless complexity

Reply:
I'm not aware of any but AFAICS the standard does not specify this. Of all the added stuff this one is so straightforward and sane-defaulted that I don't think it matters. I also checked and it has 0 perf. impact.

Reply:
A quick "LLM" search also seems to indicate LE is a possibility, but it's fairly vague about it.

Reply:
The standard specifies big endian, see page 110 https://caxapa.ru/thumbs/799244/MIPI_DCS_specification_v1.02.00.pdf

Reply:
You're right, cleaned up